metricbeat/module/etcd/leader/leader_integration_test.go

```diff
@@ -23,7 +23,7 @@ package leader
 import (
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	// "github.com/stretchr/testify/assert"
```

Comment:
Why is this and assert calls commented in this file?

Reply:
The change requested to the `leader` metricset makes it that `leader` events are triggered only if the `etcd` instance has more than 1 follower. I had a chat with @sayden and we agreed that tests can be removed for this special case.

Reply:
Then let's remove it. Having this commented in the code is very confusing.

Reply:
Remove the commented lines or the integration tests?

Reply:
@gpop63 My comment is about commented lines in this particular file. Not sure why we're talking about integration tests, sorry. Perhaps you replied to a wrong comment here?